MetriCam2.Controls/CameraExplorerControl.cs

```diff
@@ -117,13 +115,10 @@ public void RefreshAvailableCameras()
                 listViewItem.Name = cam.GetType().ToString();
                 listViewItem.Cam = cam;
 
-                Bitmap bmp = ScaleBitmap(cam.CameraIcon, IMAGE_SIZE_LARGE);
-                imageListLarge.Images.Add(cam.GetType().ToString(), (Bitmap)bmp.Clone());
-                imageListSmall.Images.Add(cam.GetType().ToString(), ScaleBitmap(cam.CameraIcon, IMAGE_SIZE_SMALL));
+                imageListSmall.Images.Add(cam.GetType().ToString(), cam.CameraIcon);
 
-                OverlayBitmap(bmp, Properties.Resources.greenDot);
-                imageListLarge.Images.Add(cam.GetType().ToString() + "_C", bmp);
-                imageListSmall.Images.Add(cam.GetType().ToString() + "_C", ScaleBitmap(bmp, IMAGE_SIZE_SMALL));
+                // FIXME: indicate that the camera is connected
```

Comment:
Creating the overlay _may_ work similar to this (untested):
```
public Icon AddIconOverlay(Icon originalIcon, Icon overlay)
{
    Icon result = (Icon)originalIcon.Clone();
    Graphics canvas = Graphics.FromImage(result);
    canvas.DrawIcon(overlay, 0, 0); // there's also an overload taking a Rectangle
    canvas.Save();
    return result;
}
```
Adapted from https://stackoverflow.com/questions/2599778/how-to-merge-two-icons-together-overlay-one-icon-on-top-of-another
I don't know how it will handle the different resolutions in an `Icon`.

Reply:
Oh, it's totally worth having a look at the `Icon` c'tors: https://msdn.microsoft.com/en-us/library/system.drawing.icon(v=vs.110).aspx
I would try what happens when you create a larger `Icon` from `CameraIcon`. If that doesn't work you have to load `CameraIcon` from the resource stream yourself (like in the `WebCam`).

Reply:
I'm not familiar with Windows Forms that much, but using WPF the preferred way of doing this would be imo to have 2 controls stacked on top of each other and not manipulating image data.
I'll try your solution for now.